README.md

```diff
@@ -1,3 +1,20 @@
-# MGMCo Handlebars helpers
+# MGMCo Handlebars helpers [![Build Status](https://travis-ci.org/mgmco/handlebar-helpers.svg?branch=master)](https://travis-ci.org/mgmco/handlebar-helpers)
```

Comment:
I nice, a button! It's failing because I create a PR?

Reply:
It's failing because the current master has no tests defined at the moment. Jest chokes when there are no tests specified.